abci/example/kvstore/kvstore.go

```diff
@@ -132,10 +132,10 @@ func (app *Application) CheckTx(_ context.Context, req *types.RequestCheckTx) (*
 	if isValidatorTx(req.Tx) {
 		if _, _, _, err := parseValidatorTx(req.Tx); err != nil {
 			//nolint:nilerr
-			return &types.ResponseCheckTx{Code: CodeTypeInvalidTxFormat}, nil
```

Comment:
maybe also include the error as part of the logs: `fmt.Sprintf("invalid-tx-format: %w")`

Reply:
addressed in this commit https://github.com/celestiaorg/celestia-core/pull/2286/commits/44c58ac9666f4207b84a4a694356b2845293b52a